### PR TITLE
Server client connection

### PR DESCRIPTION
**Pull request by Radhe2911**
Bug for image type fixed

### CODE REVIEW

File: server/src/DataTypes/dataTypes.ts [hunk 1]
```diff
@@ -3,4 +3,23 @@ import { ObjectId } from "mongodb";
...
+    namespace Express {
+        interface Request {
+            user: jwtCredential;
+        }
+    }
+}
+
+export interface fileWithTempPath {
+    tempFilePath: string,
+    [key: string]: any
```
Comment (tw-vineet) [inline server/src/DataTypes/dataTypes.ts]: why using `any`

File: server/src/DataTypes/dataTypes.ts [hunk 2]
```diff
@@ -3,4 +3,23 @@ import { ObjectId } from "mongodb";
...
+}
+
+export interface UploadedImage {
+    public_id: string;
+    secure_url: string;
+    [key: string]: any;
```
Comment (tw-vineet) [inline server/src/DataTypes/dataTypes.ts]: please remove any in hole code

File: server/src/controllers/userController.ts [hunk 3]
```diff
@@ -1,14 +1,63 @@
...
+        const userDetails = await getUserDetails(userId, next);
+        res.status(200).json({
+            status: true,
+            statusCode: 200,
+            data: userDetails,
+            message: "User details",
+        });
     } catch (error) {
-        console.log("errorrrr", error);
+        next(error)
```
Comment (tw-vineet) [inline server/src/controllers/userController.ts:22]: do not use console.log use 
`logger`

File: server/src/validations/validation.ts [hunk 4]
```diff
@@ -108,4 +116,85 @@ export const loginValidation = async (req: Request, res: Response, next: NextFun
     } catch (error) {
         console.log(error);
```
Comment (tw-vineet) [inline server/src/validations/validation.ts]: remove console.log

File: server/src/models/Users.ts [hunk 5]
```diff
@@ -69,6 +71,14 @@ export const getUserDetailsByEmail = async (email: string, next: NextFunction) =
         return response;
     } catch (error) {
         next(error);
-        console.log("error", error);
```
Comment (tw-vineet) [inline server/src/models/Users.ts:72]: remove console.log

File: server/src/models/Users.ts [hunk 6]
```diff
@@ -42,25 +47,22 @@ const User = mongoose.model("user", userSchema);
...
-export const createUser = async (values: Record<string, string>, next: NextFunction) => {
+export const createUser = async (data: Record<string, string>, next: NextFunction) => {
     try {
-        const newUser = new User(values)
+        const newUser = new User(data)
         const response = await newUser.save();
         return response;
     } catch (error) {
         next(error);
-        console.log("error", error);
```
Comment (tw-dev-alfaiz) [inline server/src/models/Users.ts:53]: replace console log in catch block with throw

File: server/src/validations/validation.ts (hunk 4 above)
Comment (Radhe2911) [reply server/src/validations/validation.ts]: Removed

File: server/src/controllers/userController.ts (hunk 3 above)
Comment (Radhe2911) [reply server/src/controllers/userController.ts:22]: done

File: server/src/models/Users.ts (hunk 5 above)
Comment (Radhe2911) [reply server/src/models/Users.ts:72]: done

**Pull request merged**